zbus/src/connection/mod.rs

```diff
@@ -719,8 +719,8 @@ impl Connection {
                                     // else the name was released in the meantime. :shrug:
                                 }
                                 Ok(_) => (),
-                                Err(e) => {
-                                    warning!("Failed to parse `NameAcquired` signal: {}", e)
+                                Err(_e) => {
+                                    warning!("Failed to parse `NameAcquired` signal: {}", _e)
```

Comment:
The task is to not have to do this at all. The caller of the macros, should not need to worry about the variables being unused.